### ISSUE TITLE
Providing argument "--stops-without-level-id" without "levels.txt"

### ISSUE THREAD

**Issue by langbein-daniel** [public-transport/gtfs-via-postgres#43]
The description of the argument `--stops-without-level-id` says: "Default if levels.txt has not been provided."

However, if I use this command-line switch on a GTFS feed without the `levels.txt` file, I get the following error:

```
Error: task "levels" is not defined
    at sequence (/app/node_modules/sequencify/index.js:14:9)
    at sequence (/app/node_modules/sequencify/index.js:38:5)
    at sequence (/app/node_modules/sequencify/index.js:38:5)
    at convertGtfsToSql (/app/index.js:137:2)
    at convertGtfsToSql.next (<anonymous>)
    at pumpToNode (node:internal/streams/pipeline:133:22)
    at pipelineImpl (node:internal/streams/pipeline:373:9)
    at pipeline (node:internal/streams/pipeline:183:10)
    at Object.<anonymous> (/app/cli.js:158:1)
    at Module._compile (node:internal/modules/cjs/loader:1257:14) {
  missingTask: 'levels',
  taskList: [
    undefined, undefined,
    undefined, undefined,
    undefined, undefined,
    undefined, undefined,
    undefined, undefined,
    undefined, undefined,
    undefined
  ]
}
```

I have tested this with the Finnish feed (https://traffic.navici.com/tiedostot/gtfs.zip) after removing `fare_leg_rules.txt` and with the feed of the VGN (https://www.vgn.de/opendata/GTFS.zip).

I am using the Docker image `ghcr.io/public-transport/gtfs-via-postgres:4`:

```
        "Id": "sha256:f410af4af2bd673ef92abce663a9c9489e16849dcd1b3b529f19f10601b581b8",
        "Created": "2023-07-07T17:12:25.46145955Z",
```

And this is my full command:

```shell
  sudo docker run --rm --volume ./GTFS:/gtfs \
    "${DOCKER_IMAGE}" \
    --import-metadata \
    --route-types-scheme google-extended \
    --stops-without-level-id \
    --trips-without-shape-id \
    --routes-without-agency-id \
    -- '/gtfs/*.txt'
```

**Comment by derhuerst** [comment]
Thanks for reporting.

This is a regression introduced by 08fbb52, in which I have accidentally removed the `if ('stops-without-level-id' in argv)` check.

**Comment by derhuerst** [comment]
Fixed published as `gtfs-via-postgres@4.7.1`. 🎉

**Comment by derhuerst** [comment]
> Just to make sure that I understand this option correctly: Whenever `--stops-without-level-id` is given, it is OK if stops have no `level_id`. And if a feed has no levels.txt, one does not need to explicitly specify this argument as it will be done automatically.

Exactly, that's the intention.

I don't think this is a very intuitive API. Proposals on how to improve its ergonomics are welcome!

**Comment by langbein-daniel** [comment]
> I cannot reproduce this problem: thinking

The difference is that I did explicitly specify `-stops-without-level-id`.

**Comment by langbein-daniel** [comment]
> > Just to make sure that I understand this option correctly: Whenever `--stops-without-level-id` is given, it is OK if stops have no `level_id`. And if a feed has no levels.txt, one does not need to explicitly specify this argument as it will be done automatically.
> 
> Exactly, that's the intention.

Shouldn't it be

```js
if ('stops-without-level-id' in flags) { 
 	opt.stopsWithoutLevelId = true
} 
```

then?

Because if stops.txt does not exist, we set it to true in https://github.com/public-transport/gtfs-via-postgres/blob/21aad0358a7ed2abe57c07cce5bd6113786f2318/index.js#L21 already.

**Comment by derhuerst** [comment]
> Shouldn't it be
> 
> ```js
> if ('stops-without-level-id' in flags) { 
>  	opt.stopsWithoutLevelId = true
> } 
> ```
> 
> then?

Yes, thank you for pointing this out. Now that I'm looking at this again, it's totally obvious. 😅

(I will change it to to `flags['stops-without-level-id']`, without the `!`, because intuitively to me it seems more in line with how Node's `util.parseArgs()` is designed.)

**Comment by derhuerst** [comment]
Second fix published as `gtfs-via-postgres@4.7.3`. 🎉
